docs/plugins/minruntime.md

```diff
@@ -0,0 +1,121 @@
+
+## Usage
+
+The minruntime plugin is configured in two ways:
+
+1. **Queue Definition**: Set minimum runtime values in Queue configurations
+2. **Plugin Arguments**: Configure default values and resolution methods in the scheduler configuration
+
+### Queue Configuration
+
+Queues can specify the following minimum runtime parameters:
+
+- `preemptMinRuntime`: Minimum runtime before a job in this queue can be preempted
+- `reclaimMinRuntime`: Minimum runtime before a job in this queue can have resources reclaimed
+
+Example Queue definition:
+
+```yaml
+apiVersion: scheduling.run.ai/v2
+kind: Queue
+metadata:
+  name: production
+spec:
+  preemptMinRuntime: "20s"
+  reclaimMinRuntime: "30s"
+```
+
+### Plugin Configuration
+
+In the scheduler configuration (`scheduler-config` ConfigMap), add the minruntime plugin with its arguments:
+
+```yaml
+tiers:
+- plugins:
+  # other plugins...
+  - name: minruntime
+    arguments:
+      defaultPreemptMinRuntime: "10m"
+      defaultReclaimMinRuntime: "10m"
+      reclaimResolveMethod: "lca"  # or "queue"
+```
+
+### Configuration Parameters
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| `defaultPreemptMinRuntime` | Default minimum runtime before preemption if not specified in queue | "0s" |
+| `defaultReclaimMinRuntime` | Default minimum runtime before resource reclamation if not specified in queue | "0s" |
+| `reclaimResolveMethod` | Method to resolve reclaim minimum runtime ("lca" or "queue") | "lca" |
+
+0s means workloads are instantly reclaimable/preemptible.
+
+## Resolution Methods
+
+### Preemption Resolution
+
+For preemption, the minimum runtime is determined by starting from the victim's queue and walking up the queue hierarchy until a `preemptMinRuntime` value is found.
+
+### Reclamation Resolution
+
+For reclamation, two methods are supported:
+
+1. **Queue-based Resolution** (`reclaimResolveMethod: "queue"`):
+   - Similar to preemption, looks at the victim's queue and walks up the hierarchy until a `reclaimMinRuntime` value is found.
+   - If no value is found at all, use the default value for the plugin.
+
+2. **LCA Resolution** (`reclaimResolveMethod: "lca"`):
+   - Identifies the Lowest Common Ancestor (LCA) in the queue hierarchy between the preemptor and victim
+   - Walks one step down from the LCA towards the victim and uses that queue's `reclaimMinRuntime` value
+   - If no value is found, moves up towards the root queue until one is found.
+   - If no value is found at all, use the default value for the plugin.
+
+The LCA method is the default method if none is specified. The purpose of the LCA method is to follow how policies might be set for queue hierarchy, allowing users in sub-queues to set min-runtime values that are honored by their siblings, whilst not affecting cousin queues.
+
+## Elastic Jobs Handling
+
+For elastic jobs (where `MinAvailable < len(PodInfos)`), the plugin:
```

Comment:
Feels like this part should be accessible for users who may not know the internals of the scheduler, and will not be familiar with "PodInfos"
```suggestion
For elastic jobs (where `MinAvailable < number of pods`), the plugin:
```